src/server/standalone.js

```diff
@@ -148,6 +148,10 @@ function StandAloneServer(gmeConfig) {
 
         __httpServer = Http.createServer(__app);
 
+        if (gmeConfig.server.timeout > 0) {
+            __httpServer.timeout = gmeConfig.server.timeout;
```

Comment:
What if this is 0? Shouldn't we still set it? According to the http docs:

>Set to 0 to disable any kind of automatic timeout behavior on incoming connections.

https://nodejs.org/api/http.html#http_server_timeout